session/pingpong/invoice_payer_test.go

```diff
@@ -689,13 +700,19 @@ func TestInvoicePayer_issueExchangeMessage(t *testing.T) {
 type mockConsumerTotalsStorage struct {
 	res     uint64
 	resLock sync.Mutex
+	bus     eventbus.Publisher
 
 	err        error
 	calledWith uint64
 }
 
-func (mcts *mockConsumerTotalsStorage) Store(providerAddress, accountantAddress string, amount uint64) error {
+func (mcts *mockConsumerTotalsStorage) Store(consumerAddress, accountantAddress string, amount uint64) error {
 	mcts.calledWith = amount
+	go mcts.bus.Publish(AppTopicGrandTotalChanged, AppEventGrandTotalChanged{
+		Current:      amount,
+		AccountantID: identity.FromAddress(accountantAddress),
```

Comment:
`Identity` struct is construct of local MYST accounts, no point use this for `accountantAddress`

Reply:
There's a point, they format the addresses differently therefore all the addresses should be stored in identity format.